tools/wake/main.cpp

```diff
@@ -333,6 +344,27 @@ int main(int argc, char **argv) {
   }
   wcl::log::subscribe(std::make_unique<JsonSubscriber>(std::move(*res)));
 
+  // Bulk logging
+  const char *bulk_dir = getenv("WAKE_BULK_LOGGING_DIR");
```

Comment:
yeah, the fact that we have this in two places makes me feel more strongly about using WakeConfig

Reply:
I can put this in WakeConfig, I probably should have but I was being lazy about it

Reply:
added